### ISSUE TITLE
snapshot create from task in CLI and SDK interface

### ISSUE THREAD

**Issue by shabazpatel** [datmo/datmo#84]
CLI interface
```
datmo snapshot --task-id <task_id>
```

SDK interface
```
import datmo
task_dict = {"command":"python script.py"}
task_id = datmo.task.run(task_dict)
datmo.snapshot.create(task_id=task_id)
```

**Comment by asampat3090** [comment]
this interface makes sense. 

Currently in the SDK there is already a `create_from_task` function that is used separately (https://github.com/datmo/datmo/blob/master/datmo/snapshot.py#L172-L206)

If we would like to merge these functions into 1 then, we will need to make sure there is validation and checks for mutually exclusive arguments

**Comment by shabazpatel** [comment]
@asampat3090 https://github.com/datmo/datmo/blob/master/datmo/core/controller/snapshot.py#L226-L232 
this has after snapshot id being given back rather than before snapshot id (for scripts)
```
return self.dal.snapshot.update({
            "id": task_obj.after_snapshot_id,
            "message": message,
            "stats": task_obj.results,
            "visible": True
        })
```

**Comment by asampat3090** [comment]
From the last comment, actually, we do not allow non-completed tasks to be snapshots for the time being which means we would never have `after_snapshot_id` be None. if we want to change that we can reopen a new issue regarding that. 

For now as per PR #97, we are pulling from the after_snapshot_id, updating the values, and then making it visible to the end user.

**Comment by asampat3090** [comment]
merged PR #97
